### PR TITLE
Adds a Remind Command

### PR DESCRIPTION
**Pull request by Bacon-Fixation**
Ready for Review (no rush)
Addressing request #556

command is only available to administrators of the discord server 
with options to mention other roles or users

image

- Caveat
if you are using Heroku, the longest timer that will work is 24 hours because of their daily reboots

Much Love
-Bacon

### CODE REVIEW

**Comment by meiaihara06** [comment]
sorry for late reply. I'm busy with exams right now :D

1- does not work if remind description is more than 1 word
image

2- it would be nice to have a command for list of reminders ( !reminders )

**Comment by Bacon-Fixation** [comment]
if it is more than one word it needs "" around it

`!remind "hello world" 5 m`
image

Much Love
-Bacon

**Comment by meiaihara06** [comment]
> If it is more than one word it needs "" around it

oh, thanks!

it seems it works properly 
thank you so much

**Comment by Bacon-Fixation** [comment]
i added a multi-word example to the command,
to help with that confusion


Much Love
-Bacon

**Review by galnir — CHANGES_REQUESTED** [review]
There should be a limit on how many reminders can be set per user (5 should be enough imo)

**Comment by Bacon-Fixation** [comment]
Working on that and the ability to remove them maybe

Much Love
-Bacon

**Comment by Bacon-Fixation** [comment]
I apologize for delays <3
I didn't find a smooth way to delete old timers yet (will make a new PR for that)

Much Love
-Bacon

File: commands/guild/remind.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+          default: 'author',
+          error: ':x: Please try again with a valid role or user.'
+        }
+      ]
+    });
+  }
+
+  run(message, { reminderMessage, number, unit, mention }) {
+    // Create if not present
+    message.member.reminders ? null : (message.member.reminders = 0);
```
Comment (galnir) [inline commands/guild/remind.js:59]: Where does .reminders property come from? Did you extend GuildMember?

Comment (Bacon-Fixation) [reply commands/guild/remind.js:59]: it let me add it in the ternary :D

**Review by galnir — APPROVED** [review]
I've added the ability to delete reminders. 
Tested and didn't run into issues

**Pull request merged**